iot-hub/Samples/common/DeviceStreamingCommon.cs

```diff
@@ -22,7 +22,7 @@ public static async Task<ClientWebSocket> GetStreamingClientAsync(Uri url, strin
             var wsClient = new ClientWebSocket();
             wsClient.Options.SetRequestHeader("Authorization", "Bearer " + authorizationToken);
 
-            await wsClient.ConnectAsync(url, cancellationToken).ConfigureAwait(false);
+            await wsClient.ConnectAsync(url, cancellationToken);
```

Comment:
For samples, we'll leave off the ConfigureAwait(false) call, to match the advice we got in the mono repo.